go/oasis-test-runner/oasis/oasis.go

```diff
@@ -627,9 +642,9 @@ func (net *Network) startOasisNode(
 			tendermintDebugAddrBookLenient().
 			tendermintDebugAllowDuplicateIP()
 	}
-	if net.cfg.UseAbstractGrpcSockets {
+	if net.cfg.UseCustomGrpcSocketPath {
 		extraArgs = extraArgs.debugDontBlameOasis()
```

Comment:
This is already set in `extraArgs` AFAIK.

Reply:
`UseCustomGrpcSocketPath` needs to be dontblamed. To avoid duplicate dontblameoasis flags, I added check in `debugDontBlameOasis()` to ignore it, if it  has already been provided before.